thesslstore_plugin.php

```diff
@@ -198,10 +198,18 @@ private function order_synchronization(){
 
         $order_query_resp = $api->order_query($order_query_request);
 
-        $services = $this->Services->getList();
+       $sub_query_sql = "SELECT `service_id` FROM `service_fields` WHERE `key` = 'thesslstore_fqdn' GROUP BY `service_id`";
```

Comment:
It'd be great if you could use the Record component to generate this query rather than writing it yourself. See [Record subqueries](https://github.com/phillipsdata/minphp-record#subqueries)

Reply:
updated in additional commits